frontend/public/locales/en/public.json

```
@@ -412,12 +412,15 @@
   "Operators": "Operators",
   "Operator status": "Operator status",
   "Cluster inventory": "Cluster inventory",
+  "To improve resource utilization you can assign more memory to the VM than the node's physical memory. Please note it requires careful management and monitoring to avoid performance issues and system instability": "To improve resource utilization you can assign more memory to the VM than the node's physical memory. Please note it requires careful management and monitoring to avoid performance issues and system instability",
```

```suggestion
  "To improve resource utilization you can assign more memory to the VM than the node's physical memory. Please note it requires careful management and monitoring to avoid performance issues and system instability": "To improve resource utilization you can assign more memory to the VM than the node's physical memory. It requires careful management and monitoring to avoid performance issues and system instability",
```
We avoid terms of politeness, such as please and thank you in the docs. I would suggest avoiding here as well if possible.